src/WinRT.Runtime/Marshalers.cs

```diff
@@ -1088,7 +1114,7 @@ public void Dispose()
                 {
                     foreach (var marshaler in _marshalers)
                     {
-                        Marshaler<T>.DisposeMarshaler(marshaler);
+                        MarshalNonBlittable<T>.DisposeMarshaler(marshaler);

```

Comment:
Can you maybe leave a comment to explain why it's guaranteed that `T` isn't blittable here? It's not obvious 🤔